packages/react-scripts/config/webpack.config.js

```diff
@@ -685,9 +685,8 @@ module.exports = function(webpackEnv) {
         new WorkboxWebpackPlugin.GenerateSW({
           clientsClaim: true,
           exclude: [/\.map$/, /asset-manifest\.json$/],
-          importWorkboxFrom: 'cdn',
           navigateFallback: paths.publicUrlOrPath + 'index.html',
-          navigateFallbackBlacklist: [
+          navigateFallbackDenylist: [
```

Comment:
[deprecated as part of the V5 breaking changes](https://github.com/GoogleChrome/workbox/releases/tag/v5.0.0)